v3.0/tidb-in-kubernetes/deploy/gcp-gke.md

```diff
@@ -55,16 +55,15 @@ gcloud services enable container.googleapis.com
 
 ### Configure Terraform
 
-The terraform script expects three environment variables. You can let Terraform prompt you for them, or `export` them in the `~/.bash_profile` file ahead of time. The required environment variables are:
+The terraform script expects three variables to be provided by the user. You can let Terraform prompt you for them, or define them in a `.tfvars` file of your choice. The three variables are:
 
-* `TF_VAR_GCP_CREDENTIALS_PATH`: Path to a valid GCP credentials file.
+* `GCP_CREDENTIALS_PATH`: Path to a valid GCP credentials file.
     - It is recommended to create a new service account to be used by Terraform. See [this page](https://cloud.google.com/iam/docs/creating-managing-service-accounts) to create a service account and grant `Project Editor` role to it.
```

Comment:
Maybe we should change this now that we have a script that automatically does this? Right now the user might start doing this here instead of just running the ./create-service-account.sh script below.

Reply:
I guess it depends what level of terraform  and GCP experience we're targeting. The steps to populate them come immediately afterwards, so a totally new user can follow those, but a more experienced user can see the three variables and fill them in on their own, if they choose. Open to suggestions on this.

Reply:
@gregwebs let me know what you think about the updated section